packages/notification-controller/src/NotificationController.test.ts

```diff
@@ -135,7 +160,9 @@ describe('NotificationController', () => {
     expect(
       // TODO: Either fix this lint violation or explain why it's necessary to ignore.
       // eslint-disable-next-line @typescript-eslint/await-thenable
-      await unrestricted.call('NotificationController:show', origin, message),
+      await unrestricted.call('NotificationController:show', origin, {
+        message,
+      }),
```

Comment:
Nit: can we remove the TODO and async here too
```suggestion
      unrestricted.call('NotificationController:show', origin, {
        message,
      }),
```